### PR TITLE
Add prod docker images. Add Travis CI setup

### PR DESCRIPTION
**Pull request by JackKarichkovskiy**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+before_install:
+  - docker build -t ekarychkovsky/team-balance-app-server-test -f ./server/Dockerfile.test ./server
+
+script:
+  - docker run ekarychkovsky/team-balance-app-server-test
+#    - docker build -f ./client/Dockerfile.ci ./client
+
+after_success:
+  - docker build -t ekarychkovsky/team-balance-app-client ./client
```
Comment (andrew-kuzmin) [inline .travis.yml]: why ekarychkovksy?

File: docker-compose.yml [hunk 2]
```diff
@@ -6,10 +6,11 @@ services:
       - POSTGRES_PASSWORD=root
       - POSTGRES_USER=postgres
       - POSTGRES_DB=teambalance
```
Comment (andrew-kuzmin) [inline docker-compose.yml]: secure?

File: server/src/main/java/com/phoenixoft/teambalanceapp/config/BeanConfig.java [hunk 3]
```diff
@@ -7,7 +7,7 @@
 import org.springframework.validation.beanvalidation.LocalValidatorFactoryBean;
 
 @Configuration
-public class BeanConfiguration {
+public class BeanConfig {
```
Comment (andrew-kuzmin) [inline server/src/main/java/com/phoenixoft/teambalanceapp/config/BeanConfig.java]: Hope now your life became easier :)

File: .travis.yml (hunk 1 above)
Comment (JackKarichkovskiy) [reply .travis.yml]: Sorry, Haven't got time to create DockerHub account for our organization. So used my personal account for it.
Will create ticket for it.

File: docker-compose.yml (hunk 2 above)
Comment (JackKarichkovskiy) [reply docker-compose.yml]: It's for local use, why not

File: server/src/main/java/com/phoenixoft/teambalanceapp/config/BeanConfig.java (hunk 3 above)
Comment (JackKarichkovskiy) [reply server/src/main/java/com/phoenixoft/teambalanceapp/config/BeanConfig.java]: S-Standarts

**Pull request merged**